lib/_http_outgoing.js

```diff
@@ -918,9 +918,8 @@ function(err, event) {
   this.destroy(err);
 };
 
-OutgoingMessage.validateHeaderName = validateHeaderName;
```

Comment:
I guess this makes this patch semver-major?

Reply:
it was introduced a week ago, and was decided to be done better as it is featured now.
It was also undocumented.

If that's a problem - then I can leave that line.
LMK how to proceed.